source/extensions/http/header_validators/envoy_default/header_validator.h

```diff
@@ -152,7 +150,8 @@ class HeaderValidator : public ::Envoy::Http::HeaderValidator {
   /*
    * Common method for validating request or response trailers.
    */
-  TrailerValidationResult validateTrailers(::Envoy::Http::HeaderMap& trailers);
+  ValidationResult validateTrailers(const ::Envoy::Http::HeaderMap& trailers);
+  void sanitizeHeadersWithUnderscores(::Envoy::Http::HeaderMap& header_map);
```

Comment:
Can this be commented since it's not in the interface

Reply:
Done